apps/demo/src/plugin-demos/wcg-core.ts

```diff
@@ -0,0 +1,36 @@
+import { Observable, EventData, Page } from '@nativescript/core';
+import { DemoSharedWcgCore } from '@demo/shared';
+import { atob, btoa } from '@nativescript/wcg-core';
+
+export function navigatingTo(args: EventData) {
+  const page = <Page>args.object;
+  page.bindingContext = new DemoModel();
+}
+
+const txt = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent ac tortor congue, pharetra elit a, volutpat erat. Etiam vel ornare quam. Integer vitae sapien malesuada tellus iaculis ultrices. In hac habitasse platea dictumst. Nulla enim magna, blandit vel massa ac, bibendum viverra tortor. Nam accumsan mi eu risus pretium sollicitudin vitae ut nibh. Vivamus lorem diam, sagittis vel leo vel, pharetra aliquam ex. Aenean sollicitudin fringilla ex. Donec malesuada justo enim. Fusce elementum velit quis quam varius, sit amet feugiat sapien faucibus. Vestibulum nec tincidunt nibh, sit amet sagittis tellus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae;
```

Comment:
Prevent the poor TypeScript compiler from having to keep this whole long string in its mind as a constant:
```suggestion
const txt: string = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent ac tortor congue, pharetra elit a, volutpat erat. Etiam vel ornare quam. Integer vitae sapien malesuada tellus iaculis ultrices. In hac habitasse platea dictumst. Nulla enim magna, blandit vel massa ac, bibendum viverra tortor. Nam accumsan mi eu risus pretium sollicitudin vitae ut nibh. Vivamus lorem diam, sagittis vel leo vel, pharetra aliquam ex. Aenean sollicitudin fringilla ex. Donec malesuada justo enim. Fusce elementum velit quis quam varius, sit amet feugiat sapien faucibus. Vestibulum nec tincidunt nibh, sit amet sagittis tellus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae;
```